cypress/TestCases/FireboltCertification/UserInterest.feature

```diff
@@ -0,0 +1,113 @@
+            | Scenario                                                | GetParam                                                     |
+            | Empty param                                             | without any params                                           |
+            | Without Interest type                                   | without interest type                                        |
+            | Invalid Interest type - integer params                  | with numeric interestType                                    |
+            | Invalid Interest type - test params                     | with string interestType                                     |
+            | Invalid Interest type - boolean params                  | with boolean interestType                                    |
+            | Invalid channelType in channel entity                   | with invalid channelType in channel entity                   |
+            | Boolean channelType in channel entity                   | with boolean channelType in channel entity                   |
+            | Integer channelType in channel entity                   | with integer channelType in channel entity                   |
+            | Invalid programType in program entity                   | with invalid programType in program entity                   |
+            | Boolean programType in program entity                   | with boolean programType in program entity                   |
+            | Integer programType in program entity                   | with integer programType in program entity                   |
+            | Invalid musicType in music entity                       | with invalid musicType in music entity                       |
+            | Boolean musicType in music entity                       | with boolean musicType in music entity                       |
+            | Integer musicType in music entity                       | with integer musicType in music entity                       |
+            | Invalid seriesId in program episode entity              | with invalid seriesId in program episode entity              |
+            | Invalid seasonId in program episode entity              | with invalid seasonId in program episode entity              |
+            | Invalid seriesId and seasonId in program episode entity | with invalid seriesId and seasonId in program episode entity |
+            | Invalid seriesId in program season entity               | with invalid seriesId in program season entity               |
+            | Invalid programType for program entity with seriesId    | with invalid programType for program entity with seriesId    |
+            | Invalid programType for program entity with seasonId    | with invalid programType for program entity with seasonId    |
+
+    @coreSDK @sdk @userinterest
+    Scenario Outline: Content.requestUserInterest - Positive Scenario: with type <Scenario>
```

Comment:
wouldn't it be better if we can differentiate between the 2 types of scenarios by adding them in scenario names itself ?
User Interest from an in-app UX
User Interest from a platform UX

Reply:
Agree with that

Reply:
added